server/test/services/program/ProgramDefinitionTest.java

```diff
@@ -1359,23 +1334,6 @@ public void serializeThenDeserialize_allFieldsPresent() throws JsonProcessingExc
                     /* addressCorrectionEnabled= */ false))
             .build();
 
-    StatusDefinitions.Status approvedStatus =
```

Comment:
**suggestion:** We should still test this, but with the new status logic to make sure it localizes appropriately. Does that exist somewhere else?

Reply:
Localizations are tested as part of StatusService tests.